static/app/components/forms/datePickerField.tsx

```diff
@@ -50,20 +47,12 @@ export default function DatePickerField(props: DatePickerFieldProps) {
 
                 {isOpen && (
                   <CalendarMenu {...getMenuProps()}>
-                    <Suspense
-                      fallback={
-                        <Placeholder width="332px" height="282px">
-                          <LoadingIndicator />
-                        </Placeholder>
+                    <DatePicker
+                      date={inputValue}
+                      onChange={date =>
+                        handleChangeDate(onChange, onBlur, date, actions.close)
                       }
-                    >
-                      <Calendar
-                        date={inputValue}
-                        onChange={date =>
-                          handleChangeDate(onChange, onBlur, date, actions.close)
-                        }
-                      />
-                    </Suspense>
+                    />
                   </CalendarMenu>
```

Comment:
This is probably outside the scope of this PR, but we should consider using the new `Overlay` component & `useOverlay` hook instead of just `DropdownMenu`. That way we can avoid positioning issues like this:

<img width="567" alt="image" src="https://user-images.githubusercontent.com/44172267/182470258-87eaad89-8d9c-4a54-af5f-e7f05dcb22ac.png">

Reply:
Good point - I am loving the move to `useOverlay` btw!